avocado/utils/disk.py

```diff
@@ -13,10 +13,11 @@
 #
 # client/base_utils.py
 #
-# Copyright: 2018 IBM
+# Copyright: 2023 IBM
 # Authors : Praveen K Pandey <praveen@linux.vnet.ibm.com>
 #         : Narasimhan V <sim@linux.vnet.ibm.com>
 #         : Naresh Bannoth <nbannoth@linux.vnet.ibm.com>
+#         : Maram Srimannarayana Murthy <msmurthy@linux.vnet.ibm.com>
```

Comment:
ah, I missed it. I did not feel we needed to add an author and copywriter   change

Reply:
Hi Praveen,
Do you want me to do following
1. Keep copyright as 2018
2. Remove author name in line 20.

Reply:
yes correct